storybook/storybook-css/src/docs/breakpoint.stories.mdx

```diff
@@ -0,0 +1,9 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Designrichtlijnen/Breekpunt" />
+
+# Breekpunt
+
+We hanteren 1 breekpunt waar het ontwerp van een website zich aanpast voor een goede gebruikerservaring op verschillend formaat apparaten: 854px breed.
+
+In _css_ werken we _mobile first_: eerst maken we het ontwerp onder het breekpunt en vervolgens gebruiken we een _media query_ om het ontwerp boven het breekpunt te maken.
```

Comment:
Mobile first would also indicate that you mainly use min-width media queries, doing this would help a lot preventing confusion. Maybe some code example would help:

```CSS
@media (min-width: 854px) {
    /* Your desktop styling */
}
```

En misschien wat het token voor breakpoint is?